.github/workflows/release-drafter.yml

```
@@ -0,0 +1,68 @@
+name-template: '$RESOLVED_VERSION'
```

actionlint (1.7.4)

unexpected key "name-template" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)